papers/oliver_beckstein/oliver_beckstein.rst

```diff
@@ -93,6 +93,32 @@ The user interface and modular design work equally well in complex scripted work
 MDAnalysis is available in source form under the GNU General Public License v2 from GitHub https://github.com/MDAnalysis/mdanalysis and PyPi;  conda packages are also available. The documentation is extensive http://docs.mdanalysis.org including an introductory tutorial http://www.mdanalysis.org/MDAnalysisTutorial/ and a very friendly and welcoming user and developer community. 
 
 
+New data Structures
+-------------------
+
+Originally MDAnalysis followed a strict object oriented approach with a separate instance of an Atom object for each particle in the simulation data.
+The AtomGroup then simply stored its contents as a list of these Atom instances.
+With simulation data commonly containing 10\ :sup:`6` particles
+this solution did not scale well and so
+recently this design was overhauled to improve the scalability of MDAnalysis.
+
+With all Atoms having the same property fields it was possible to store this information as a series of NumPy arrays,
+with a single array for each property.
+The AtomGroup can then store its contents as an array of integers
+which represent the address in the property arrays which correspond to the data for the stored Atoms.
```

Comment:
> Because all Atoms have the same properties (ie charge, mass) it **is** possible to store this information as a single NumPy array per property. Now an AtomGroup only needs keep track which entry in the property arrays corresponds to which Atom, this can be done with a simple integer array.

My semi successful help the text flow better to read. The original is very stiff and hard to read.